hasnaat-iftikhar.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Hasnaat Iftikhar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "FullStack developer and Elevating my skills in Cloud!",
+  "avatar": "https://lh3.googleusercontent.com/oL7jB41g1IoOM1VCwAckG6TVF-cPZkzwk0kRK7aJ47IrEvXj1elbek964WTMhICi-mpRHBAWw34o4F62VzeozfBd6zKOVMHrtAD-YMVwPrSZuv17uQKZ28pYOMNMIx1JWuB3Gn9PMcoD__Bu9uA6ov39JmBKKwc2vtQ7NGGh4goGtPpK-ga0aljTWfWBthR73xbt_nGf0l643csyFY0LXStEoFmZBrFOI8Pa0A5NjAFeFpsB_PRuToFAqmZAgwaKJNOvzWtPFg6eTL120VYORN0vW15Ezk9KgqbB0pFunX1ibSR--jAYFPeTzNTavT8Dnmk3pYaUjmgumLxvEKR6MrazPx_GK2R76PzByeb7PvAGGoEAb_CnCKsubbCS7zbKIPUhTumlVriMysay1psEhYAtKz3e9J_GQQOuwBBbtCoF5oykbwQeSlXwxCLib8PfIPLqSpQFbnmpChEHUlHRsVKpffWWcESNlPdJ5_3szaL5OmNPakRkpxsoAgTk7IeIRMdMMAzXLTwALLb-g23G3Rrld3RDLsRI_5xlOvAiY11WyU8vcoUx4UFEiJ2ddmEcmmhZwBliVA7HflNr9JWE-gRzZK1NglIOZm4QtXSeVDRyMGBxJPtb5w-hou9j6fFDVdmx90t2ww3aK7Vocy2OiWne97_aQdeELMzMvxkIeZZw_53181v4lYtyTSVUYiFc0EGhkJc1W1jPwJeb9glqV2-t8vO7uQDwFIocWGGP2tBvJucu8-0QpvGLYi_UhIpkREV-uAvXa1_plgDhSVT-cZxqy_REJB1KYjOzd1dNT6Z0ZpI6VgtZ9TbSVtx_zd7SBdyTblsEDmi1ZieanXgUPIc8PbcyqyoCMYr_U8I5_D8726j_zJd0anfxqxto69uN-5AXX46bONY1DQ1w6uoNcHtxAQDyyoCw4uKtULAEznz-H40gcBvlda800dBOdhbrK80Kcg35Y3se9_mT0w4D7THUxUtThEe5oYNN0n1qR8zrUVi9qL32lw=s625-no?authuser=0",
```

Comment:
use this for the avatar link: `https://github.com/Hasnaat-Iftikhar.png`

Reply:
The replaced the avatar link successfully